src/components/ActiveLearnersTable/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import TableContainer from '../../containers/TableContainer';
+import { formatTimestamp, formatPercentage } from '../../utils';
+import EnterpriseDataApiService from '../../data/services/EnterpriseDataApiService';
+
+const ActiveLearnersTable = () => {
+  const tableColumns = [
+    {
+      label: 'Email',
+      key: 'user_email',
+      columnSortable: true,
+    },
+    {
+      label: 'Course Title',
+      key: 'course_title',
+      columnSortable: true,
+    },
+    {
+      label: 'Course Price',
+      key: 'course_price',
+      columnSortable: true,
+    },
+    {
+      label: 'Start Date',
+      key: 'course_start',
+      columnSortable: true,
+    },
+    {
+      label: 'End Date',
+      key: 'course_end',
+      columnSortable: true,
+    },
+    {
+      label: 'Passed Date',
+      key: 'passed_timestamp',
+      columnSortable: true,
+    },
+    {
+      label: 'Current Grade',
+      key: 'current_grade',
+      columnSortable: true,
+    },
+    {
+      label: 'Last Activity Date',
+      key: 'last_activity_date',
+      columnSortable: true,
+    },
+  ];
+
+
+  const formatPassedTimestamp = (timestamp) => {
```

Comment:
Because this function is now shared between more than one table, it may be a good idea to move it to `utils.js` as a utility function.